src/main/webapp/scripts/my-trips.js

```diff
@@ -281,9 +380,12 @@ async function parseAndRenderHotelResults(json, centerPoint) {
                   <p>${formatted_address}</p>
                 </div>
                 <div class="card-action center">
-                  <button 
+                  <button
                     class="btn indigo" 
-                    onClick="saveTrip('${place_id}', '${photo_ref}', '${name}')"
+                    onClick="saveTrip('${place_id}', 
+                                      '${photo_ref}', 
+                                      '${name.replace(/'/g, "\\'")}',
```

Comment:
What does the string within replace() mean? Is it regex?

Reply:
Yep, this is regex to escape all singlequotes in hotel names (e.g. Fisherman's Hotel) since we are delimiting each argument here using singlequotes.